sap/cli/strust.py

```diff
@@ -0,0 +1,73 @@
+"""
+Uploads X.509 Base64 certificates into SAP to enable SSL peer verification
+of remote servers
+"""
+
+import logging
+from collections import namedtuple
+
+import sap.cli.core
+from sap.errors import SAPCliError
+from sap.rfc.strust import SSLCertStorage
+
+Identity = namedtuple('Identity', ['pse_context', 'pse_applic'])
+
+CLIENT_ANONYMOUS = 'client_anonymous'
+CLIENT_STANDART = 'client_standart'
+
+IDENTITY_MAPPING = {
+    CLIENT_ANONYMOUS: Identity('SSLC', 'ANONYM'),
+    CLIENT_STANDART: Identity('SSLC', 'DEFAULT')
+}
```

Comment:
I need this in `sap/rfc/strust.py` because I consider it a part of sapcli's API for strust which I need to use in my Ansible modules - I don't want to do `import sap.cli.strust` because I don't want to rely on FrontEnd.